README_LINUX.md

```diff
@@ -1,211 +1,203 @@
+You can actually name this whatever you want, allowing you to have multiple independent build directories. If your SuperCollider source is also a git repository, the `.gitignore` file is configured to ignore files of the form `build*`.
 
-    http://supercollider.github.io/development/building
+### Step 2: Set CMake flags
 
-The minimum required version of GCC is 4.8.
+Depending on what SuperCollider components you wish to install, you can set CMake flags. You can set CMake flags on the command line using `cmake -DKEY=value ..`. You can also use cmake frontends like ccmake or cmake-gui, or simply edit the `CMakeCache.txt` file. CMake flags are persistent and you only need to run these commands once each.
 
+We will cover a few important settings. There are others, which you can view with `cmake -LH ..`. We will document more of them in this README file soon.
 
-Building
---------
+#### Qt
 
- - to build supercollider with cmake, it is suggested to do out-of-tree
-builds in a specific build directory:
+If you are installing sclang with Qt and IDE, it is required to tell SuperCollider where Qt is. To do so:
 
-   ```
-   $> mkdir build
-   $> cd build
-   $> cmake -DCMAKE_PREFIX_PATH=/path/to/qt5 ..
-   ```
+    cmake -DCMAKE_PREFIX_PATH=/path/to/qt5 ..
 
-   The `..` at the end is easy to miss. Don't forget it!
+The location of `/path/to/qt5` will depend on how you installed Qt:
 
-   The location of `/path/to/qt5` will depend on how you installed Qt:
+- If you used the Launchpad PPA's described above, the path is `/opt/qt511` or `/opt/qt510` (depending on which version you installed).
+- If you downloaded Qt from the Qt website, the path is two directories down from the top-level unpacked Qt directory, in a folder called `gcc`: `Qt/5.11.0/gcc_64/` (64-bit) or `Qt/5.11.0/gcc/` (32-bit). By default, the Qt installer places `Qt/` in your home directory.
+- When newer versions of Qt become available through your Linux distribution's repositories, the directory `/usr/lib/x86_64-linux-gnu/` (64-bit) or `/usr/lib/i386-linux-gnu/` (32-bit) will work.
 
-   - If you used your Linux distribution's repositories, it will be `/usr/lib/i386-linux-gnu/` (32-bit) or `/usr/lib/x86_64-linux-gnu/` (64-bit).
-   - If you downloaded Qt from the Qt website, the path is two directories down from the top-level unpacked Qt directory: `Qt5.x.x/5.x/gcc/` (32-bit) or `Qt5.x.x/5.x/gcc_64/` (64-bit).
+If you want to build without Qt entirely, run
 
-   You can see the available build options with ```cmake -LH```.
+    cmake -DSC_QT=OFF ..
 
- - to run the build process run:
+Note: running headless SC in an X-less environment requires
+jackd without D-bus support. On Raspbian Jessie this requires
+compiling jackd rather than using the packaged version.
+Also, note that you will get errors on sclang startup from
+classes requiring Qt. A workaround and more details are
+described in on the [RPi](http://supercollider.github.io/development/building-raspberrypi) page.
```

Comment:
@snappizz a minor thing but I believe I fixed the error you're mentioning here some time ago

Reply:
great -- note removed